src/index.ts

```diff
@@ -267,8 +307,8 @@ class Oas {
         // instead of doing that we need to re-regex the server URL, this time splitting on the path parameters -- this
         // way we'll be able to extract the parameter names and match them up with the matched server that we obtained
         // above.
-        const variables = {};
-        [...server.url.matchAll(/{([-_a-zA-Z0-9[\]]+)}/g)].forEach((variable, y) => {
```

Comment:
Converted this to use `Array.from()` instead because Typescript was requiring a down-level transpiling in order to get its iterator working with the spread operator. Doesn't need that with `Array.from()`. 🤷